### ISSUE TITLE
JResolv and display multiplicity on horizontal axis

### ISSUE THREAD

**Issue by lpatiny** [cheminfo/nmrium#2838]
@jobo322 Apparently you did in the visualizer. could you add a screen snapshot here ?

**Comment by jobo322** [comment]
Yes we have some code for this. From the zones picked. We could extract the multiplicity
